lib/rules/keyframe-selector-notation/index.js

```diff
@@ -0,0 +1,110 @@
+'use strict';
+
+const report = require('../../utils/report');
+const ruleMessages = require('../../utils/ruleMessages');
+const validateOptions = require('../../utils/validateOptions');
+
+const ruleName = 'keyframe-selector-notation';
+
+const messages = ruleMessages(ruleName, {
+	expected: (selector, fixedSelector) => `Expected "${selector}" to be "${fixedSelector}"`,
+});
+
+const meta = {
+	url: 'https://stylelint.io/user-guide/rules/list/keyframe-selector-notation',
+};
+
+const INTERCHANGEABLE_KEYFRAME_SELECTORS = new Set(['0%', '100%', 'from', 'to']);
```

Comment:
This was the simplest approach, but I'm not sure if this is maintainable. Should this instead go into `keywordSets`, be the union of two sets, etc.?

Reply:
Defining the constant used only in this module seems to make sense, as the first implementation. 👍🏼